src/spdx2/agent/spdx2.php

```diff
@@ -716,7 +720,7 @@ protected function generateFileNodesByFiles($filesWithLicenses, $treeTableName)
       {
         $licenses['scanner'] = array();
       }
-      $content .= $this->renderString($this->getTemplateFile('file'),array(
+      $array = array(
```

Comment:
Please rename the variable, avoid naming to keywords.

Reply:
OK